packages/scene-composer/src/components/panels/scene-settings/SceneTagSettingsEditor.spec.tsx

```diff
@@ -98,26 +98,6 @@ describe('SceneTagSettingsEditor', () => {
     });
   });
 
-  it('should update input value when store value changed', async () => {
```

Comment:
CALLOUT
This test failed since the `tagSettings` aren't triggering a stateful update when the document is updated. Refresh behavior works fine & I don't believe this impacts existing prod state, however, would like to confirm that this is non-intrusive to dynamic scene work. 

Would ideally like to bring this back if possible.

Reply:
Yeah, this is a dumb test.